schema/filter_partners.js

```diff
@@ -13,7 +13,7 @@ import {
 const FilterPartners = {
   type: FilterPartnersType,
   description: 'Partners Elastic Search results',
-  args: _.create(Partners.args, {
+  args: _.extend(Partners.args, {
```

Comment:
https://lodash.com/docs#assignIn

You don't want to be mutating the existing Partner.args hash—

Rather use:

```_.assign({}, Partners.args, { ...yourargs... })
```